### PR TITLE
Changes for CP4D

### PR DESCRIPTION
**Pull request by caroazad**
Added changes to install cp4d for https://jsw.ibm.com/browse/MASCORE-1287
Screenshot 2024-06-01 at 1 32 04 AM

### CODE REVIEW

File: instance-applications/080-ibm-cp4d/templates/00-ibm-cp4d-prereqs.yaml [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+            - -c
+            - |
+
+              set -e
+
+              echo
+              echo "================================================================================"
+              echo "Required Cleanup before installing CP4D Platform operator"
+              echo "================================================================================"
+
```
Comment (whitfiea) [inline instance-applications/080-ibm-cp4d/templates/00-ibm-cp4d-prereqs.yaml:112]: What does this job do? Is it just for preparing for migrating from an older version of CP4D 4.8? If so then I don't think we need it as it is assumed that this is a new install with no previous older CP4D

File: root-applications/ibm-mas-account-root/values.yaml [hunk 2]
```diff
@@ -7,7 +7,7 @@ avp:
 
 source:
   repo_url: "https://github.com/ibm-mas/gitops"
-  revision: "poc"
+  revision: "mega"
```
Comment (whitfiea) [inline root-applications/ibm-mas-account-root/values.yaml:10]: need to change to poc

File: root-applications/ibm-mas-cluster-root/values.yaml [hunk 3]
```diff
@@ -10,7 +10,7 @@ avp:
 
 source:
   repo_url: "https://github.com/ibm-mas/gitops"
-  revision: "poc"
+  revision: "mega"
```
Comment (whitfiea) [inline root-applications/ibm-mas-cluster-root/values.yaml:13]: need to change to poc

File: root-applications/ibm-mas-instance-root/values.yaml [hunk 4]
```diff
@@ -7,7 +7,7 @@ avp:
 
 source:
   repo_url: "https://github.com/ibm-mas/gitops"
-  revision: "poc"
+  revision: "mega"
```
Comment (whitfiea) [inline root-applications/ibm-mas-instance-root/values.yaml:10]: need to change to poc

**Review by whitfiea — CHANGES_REQUESTED** [review]

File: instance-applications/080-ibm-cp4d/templates/00-ibm-cp4d-prereqs.yaml (hunk 1 above)
Comment (caroazad) [reply instance-applications/080-ibm-cp4d/templates/00-ibm-cp4d-prereqs.yaml:112]: yes, i was assuming that before I saw that common services had an older version installed in noble3 so I thought it came with one of the other installs. Should we remove it?

Comment (caroazad) [reply instance-applications/080-ibm-cp4d/templates/00-ibm-cp4d-prereqs.yaml]: removed

**Review by whitfiea — APPROVED** [review]

**Pull request merged**